docs/appendix.rst

```diff
@@ -524,76 +509,190 @@ To fix this, install the Python "wheel" package::
-use.
+This is an error that is created by the SSL library (normally OpenSSL or
+LibreSSL) and handed back up to the SSL module of Python which hands it up to
+pywbem. The error indicates that OpenSSL and the WBEM server do not agree about
+which SSL/TLS protocol level to use.
 
 Pywbem specifies SSL parameters such that the highest SSL/TLS protocol version
-is used that both the client and server support. Thus, pywbem does not put any
-additional restrictions on top of OpenSSL.
+is used that both the client and server support. Pywbem itself does not put any
+additional TLS version restrictions on top of SSL library.
 
 Debian buster includes OpenSSL 1.1.1d and increased its security settings to
-require at least TLS 1.2 (see https://stackoverflow.com/a/53065682/1424462).
-
-This error means that the WBEM server side does not yet support
-TLS 1.2 or higher or that the SSL library used by pywbem does not support
-TL 1.2.
+require at least TLS 1.2 (see `https://stackoverflow.com/a/53065682/1424462`).
 
-This issue can be corrected by:
+This issue can possibly be corrected by:
 
-1. If the current version of urllib3 is less than 2.0, update the version
-   of urllib3 (ex. ``pip install --upgrade  --upgrade-strategy eager urllib3``). Since
-   the pywbem install does not force an eager update of packages, if a valid
-   previous version of urllib3 exists it will not be upgraded to 2.0+ in the
-   reinstallation of pywbem.
+1. If the current version of urllib3 is less than 2.0 (``pip list``), update
+   the version of urllib3 (ex. ``pip install --upgrade  --upgrade-strategy
+   eager urllib3``). to a later version. This is required because pip install
+   does not force an eager update of packages; if a valid previous version (
+   i.e. any version in the range defined by the dev-requirements.txt file) of
+   urllib3 exists it will not be upgraded to 2.0+ in the reinstallation of
+   pywbem.
 
 2. If the current version of urllib3 is greater than 2.0, the previous version
    of urllib3 (ex. version 1.26.5) can be installed (ex. ``pip install
-   urllib3<2.0``)
+   urllib3<2.0``).
 
-3. Adding TLS 1.2 support to the server side (preferred) or by lowering the
-    minimum TLS level OpenSSL requires on the client
-    side (which lowers security). The latter can be done by changing the
-    ``MinProtocol`` parameter in the OpenSSL config file on the client OS
-    (typically ``/etc/ssl/openssl.cnf`` on Linux and OS-X,
-    and ``C:\OpenSSL-Win64\openssl.cnf`` on Windows).
-    At the end of the file there is::
+3. Adding TLS 1.2 support to the WBEM server side (preferred) or lowering the
+   minimum TLS level required on the client side (which lowers security). With
+   OpenSSL the latter can be done by changing the ``MinProtocol`` parameter in
+   the OpenSSL config file on the client OS (typically ``/etc/ssl/openssl.cnf``
+   on Linux and OS-X, and ``C:\OpenSSL-Win64\openssl.cnf`` on Windows).
```

Comment:
Can the minimum TLS version be lowered that way with OpenSSL >= 1.1.1? The previous text suggests that is not the case.